qatrack/qa/admin.py

```
@@ -513,15 +514,15 @@ def clean(self):
 class TestListMembershipInlineFormSet(forms.models.BaseInlineFormSet):
 
     def __init__(self, *args, **kwargs):
```

Ruff (0.15.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)